engine/crash/src/backtrace_jsweb.cpp

```diff
@@ -64,19 +63,15 @@ extern "C" void JSWriteDump(char* json_stacktrace) {
         return;
     dmCrash::g_AppState.m_PtrCount = 0;
     dmCrash::g_AppState.m_Signum = 0xDEAD;
-    dmJson::Document doc = { 0 };
-    if (dmJson::Parse(json_stacktrace, &doc) == dmJson::RESULT_OK)
+    int length = strlen(json_stacktrace);
+    uint32_t len = dmMath::Min((dmCrash::AppState::EXTRA_MAX - 1), (uint32_t)length);
+    strncpy(dmCrash::g_AppState.m_Extra, json_stacktrace, len);
+    if (g_CrashExtraInfoCallback)
```

Comment:
This was the only real other place where we used the Json parser.
 And if it didn't parse ok, we didn't store the message (which seems odd tbh)